### PR TITLE
Update extended indexes docs for GA

### LINKED ISSUE #3199: Update extended indexes docs for GA
The `extendedIndexes` feature will go into GA for Prisma 4, so will need to:

- remove all references to the Preview feature
- update the [Database features tables](https://www.prisma.io/docs/reference/database-reference/database-features#indexes)

### PR DESCRIPTION
**Pull request by keerlu**
## Describe this PR

- remove references to Preview
- add to database features table

## What issue does this fix?

Fixes #3199

### CODE REVIEW

File: content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx [hunk 1]
```diff
@@ -8,23 +8,21 @@ preview: true
 
 <TopBlock>
 
-Prisma allows configuration of database indexes, unique constraints and primary key constraints through the `extendedIndexes` preview feature in version `3.5.0` and later. This provides extended syntax options in the Prisma schema language for configuring the length of indexes (MySQL only), the sort order (all databases), the index type (PostgreSQL only) and to configure indexes as clustered or non-clustered (SQL Server only).
+Prisma allows configuration of database indexes, unique constraints and primary key constraints. This can be enabled through the `extendedIndexes` Preview feature in versions `3.5.0` and later, and is in General Availability in versions `4.0.0` and later.
```
Comment (tanberry) [inline content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx]: A couple of suggestions for a rewrite to avoid passive voice, and to be more direct:
"You can enable this through the `extendedIndexes` Preview feature..."
"Enable this through the `extendedIndexes` Preview feature..."

Also I didn't know that we are capitalizing "Preview" but I do like it and it feels necessary, to make it more obvious. We should add to the Style Guide, along with a million other things.

File: content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx [hunk 2]
```diff
@@ -8,23 +8,21 @@ preview: true
...
 
-Version `3.6.0` also introduces support for introspection and migration of full text indexes in MySQL and MongoDB through a new `@@fulltext` attribute, available through the `fullTextIndex` preview feature.
+Version `3.6.0` also introduces support for introspection and migration of full text indexes in MySQL and MongoDB through a new `@@fulltext` attribute, available through the `fullTextIndex` Preview feature.
 
 <Admonition type="warning">
 
-These changes to index configuration and full text indexes might be **breaking changes** if you have a database that is already making use of these features, so they need to be explicitly enabled via preview features.
+These changes to index configuration and full text indexes might be **breaking changes** if you have a database that is already making use of these features.
```
Comment (tanberry) [inline content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx]: Can we add one more sentence explaining why it would be a breaking change if you already have that feature enabled? I guess because new version implements the feature in a diff way?

**Review by tanberry — APPROVED** [review]

File: content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx (hunk 1 above)
Comment (andrew-walford-prisma) [reply content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx]: I suggest "with" instead of "through". I think it's clearer for second-language speakers.

Comment (andrew-walford-prisma) [reply content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx]: Do we need to capitalize preview? On a more general point, I've written stabilization notices differently (and yes, I had "preview" down as lower case), and I think moving forwards we should have a standard way of expressing it. Not necessarily my way! We write about this often enough that we could, as a group, come up with an ideal wording.

File: content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx (hunk 2 above)
Comment (andrew-walford-prisma) [inline content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx]:
```suggestion
These changes to index configuration and full text indexes might be **breaking changes** if you have a database that already uses these features.
```

File: content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx [hunk 3]
```diff
@@ -8,23 +8,21 @@ preview: true
...
 </Admonition>
 
 </TopBlock>
 
 ## Index configuration
 
-The `extendedIndexes` preview feature enables new configuration options for indexes, unique constraints and primary key constraints in Prisma.
-
-The following arguments can be specified:
+Prisma allows you to configure indexes, unique constraints and primary key constraints using the following attribute arguments:
```
Comment (andrew-walford-prisma) [inline content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx]:
```suggestion
You can configure indexes, unique constraints, and primary key constraints with the following attribute arguments:
```

File: content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx [hunk 4]
```diff
@@ -500,12 +487,12 @@ model Post {
 }
 ```
 
-On MongoDB, the `fullTextIndex` and `extendedIndexes` preview features can be combined to add fields in ascending or descending order to your full-text index:
+On MongoDB, the `@@fulltext` index attribute can be used with the `sort` argument to add fields to your full-text index in ascending or descending order. In the following example, a `@@fulltext` index is added to the `title` and `content` fields of the `Post` model, sorting the `title` field in descending order:
```
Comment (andrew-walford-prisma) [inline content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx]:
```suggestion
On MongoDB, you can use the `@@fulltext` index attribute with the `sort` argument to add fields to your full-text index in ascending or descending order. The following example adds a `@@fulltext` index to the `title` and `content` fields of the `Post` model, and sorts the `title` field in descending order:
```

**Review by andrew-walford-prisma — COMMENTED** [review]
I've made a few suggestions, but your PR otherwise LGTM!

File: content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx (hunk 1 above)
Comment (keerlu) [reply content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx]: We do capitalize 'Preview' everywhere - not sure whether I like it but that's the convention currently.

Thanks, I've rewritten to use active voice and "with" - still reads as clunky so feel free to have another crack at it/let me know what you've used elsewhere. Agree that we should come up with a wording.

File: content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx (hunk 2 above)
Comment (keerlu) [reply content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx]: Agree - I'll improve this as I write the corresponding section in the upgrade guide

File: content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx [hunk 5]
```diff
@@ -8,23 +8,21 @@ preview: true
...
+Prisma allows configuration of database indexes, unique constraints and primary key constraints. This is in General Availability in versions `4.0.0` and later. You can enable this with the `extendedIndexes` Preview feature in versions `3.5.0` and later.
 
-Version `3.6.0` also introduces support for introspection and migration of full text indexes in MySQL and MongoDB through a new `@@fulltext` attribute, available through the `fullTextIndex` preview feature.
+Version `3.6.0` also introduces support for introspection and migration of full text indexes in MySQL and MongoDB through a new `@@fulltext` attribute, available through the `fullTextIndex` Preview feature.
 
 <Admonition type="warning">
 
-These changes to index configuration and full text indexes might be **breaking changes** if you have a database that is already making use of these features, so they need to be explicitly enabled via preview features.
+These changes to index configuration and full text indexes might be **breaking changes** if you have a database that already uses these features.
```
Comment (do4gr) [inline content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx]: Only when upgrading from a pre 4.0 to 4.0 or from pre 4.0 without preview feature to pre 4.0 with preview feature. Do we generally keep around warnings around breaking changes in the docs for majors?

File: content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx [hunk 6]
```diff
@@ -8,23 +8,21 @@ preview: true
...
 </TopBlock>
 
 ## Index configuration
 
-The `extendedIndexes` preview feature enables new configuration options for indexes, unique constraints and primary key constraints in Prisma.
-
-The following arguments can be specified:
+You can configure indexes, unique constraints, and primary key constraints with the following attribute arguments:
 
 - The `length` argument is available **in MySQL only** on the `@id`, `@@id`, `@unique`, `@@unique` and `@@index` attributes in version `3.5.0` and later. It allows Prisma to support indexes and constraints on `String` and `Bytes` types.
```
Comment (do4gr) [inline content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx:27]: I'd love to not have the versions from when this was available in preview in here. Would it be fine to say available from 4.0?

File: content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx (hunk 5 above)
Comment (floelhoeffel) [reply content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx]: I would remove that here. Maybe we could link to the migration guide though?

**Comment by pimeys** [comment]
I can't comment to the PR as a change request, but the SpGist code example has the comment pointers in a wrong place (should be after the `@@index` attribute: https://deploy-preview-3235--prisma2-docs.netlify.app/docs/concepts/components/prisma-schema/indexes#space-partitioned-gist-sp-gist

**Comment by pimeys** [comment]
Did a PR against this branch, adding a few missing things https://github.com/prisma/docs/pull/3307

File: content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx (hunk 6 above)
Comment (Jolg42) [reply content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx:27]: Also, it's important to note that for this line here, it is available from `3.5.0` but the preview feature flag was also required.

File: content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx [hunk 7]
```diff
@@ -8,23 +8,21 @@ preview: true
 
 <TopBlock>
 
-Prisma allows configuration of database indexes, unique constraints and primary key constraints through the `extendedIndexes` preview feature in version `3.5.0` and later. This provides extended syntax options in the Prisma schema language for configuring the length of indexes (MySQL only), the sort order (all databases), the index type (PostgreSQL only) and to configure indexes as clustered or non-clustered (SQL Server only).
+Prisma allows configuration of database indexes, unique constraints and primary key constraints. This is in General Availability in versions `4.0.0` and later. You can enable this with the `extendedIndexes` Preview feature in versions `3.5.0` and later.
```
Comment (janpio) [inline content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx:11]: This change loses the information what you can actually configure. Fine if intentional.

File: content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx (hunk 5 above)
Comment (janpio) [reply content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx]: Yes we generally keep warnings around but make it very clear for which versions. We should not remove the warning.

This is useful as the majority of Prisma users will not actually start out with Prisma 4 when we release it, but still have to do that upgrade when they learn about this feture. This ensures they can learn about the breaking changes.

Example is this red warning block here: https://www.prisma.io/docs/concepts/components/prisma-schema/relations/referential-actions#what-are-referential-actions:~:text=If%20you%20upgrade%20from%20a%20version%20earlier%20than

So maybe something like

> If you are upgrading from a version earlier than 4.0.0, using the index configuration options [list] might lead to breaking changes. Please read our upgrade guide for Prisma 4.0.0 to learn how to handle these.

File: content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx (hunk 6 above)
Comment (janpio) [reply content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx:27]: We usually go all the way here indeed an spell out since when it was GA available, but then also add as an aside with which preview feature it was availalbe for which versions.

Example: https://www.prisma.io/docs/concepts/components/prisma-schema/relations/referential-actions#what-are-referential-actions:~:text=In%202.26.0%20and%20later%20(before%203.0.0%20with%20a%20preview%20feature%20flag%20referentialActions%2C%20afterwards%20without)%2C%20you%20can%20define%20referential%20actions%20on%20your%20relation%20fields%20in%20your%20Prisma%20schema.

This is useful as we have and will keep having loads of users on versions between 3.5.0 and 4.0.0 where the preview feature is still required.

(We have no policy yet when and if we remove these outdated preview notes)

Comment (janpio) [reply content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx:27]: (In this case it might make sense to just seaprate the preview inforamtion into a section _after_ the initial section so that this becomes easier to read as @do4gr suggested. The preview information could go in stead of where we currentl have the instrucions on how to enable the preview feature then.)

File: content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx [hunk 8]
```diff
@@ -286,9 +273,9 @@ model Example {
   value String
   //    ^ field type matching the operator class
 
+  @@index([value], type: SpGist)
   //                     ^ index type
   //       ^ using the default ops: TextOps
-  @@index([value], type: SpGist)
```
Comment (janpio) [inline content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx:291]: Huh what had happened here? That looks like it was totally wrong before.

File: content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx [hunk 9]
```diff
@@ -500,12 +487,12 @@ model Post {
 }
 ```
 
-On MongoDB, the `fullTextIndex` and `extendedIndexes` preview features can be combined to add fields in ascending or descending order to your full-text index:
+On MongoDB, you can use the `@@fulltext` index attribute with the `sort` argument to add fields to your full-text index in ascending or descending order. The following example adds a `@@fulltext` index to the `title` and `content` fields of the `Post` model, and sorts the `title` field in descending order:
```
Comment (janpio) [inline content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx]: Note sure if needed, but to be absolutely precise:
```suggestion
On MongoDB, you can use the `@@fulltext` index attribute (via the `fullTextIndex` preview feature) with the `sort` argument to add fields to your full-text index in ascending or descending order. The following example adds a `@@fulltext` index to the `title` and `content` fields of the `Post` model, and sorts the `title` field in descending order:
```

File: content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx (hunk 8 above)
Comment (pimeys) [reply content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx:291]: Yep. Found it when casually reading the pages...

Comment (pimeys) [reply content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx:291]: Maybe a merge conflict?

File: content/400-reference/200-api-reference/100-prisma-schema-reference.mdx [hunk 10]
```diff
@@ -1169,7 +1169,7 @@ Defines a single-field ID on the model.
 > @id
 > ```
 
-> **Note**: In version 3.5.0 and later, with the `"extendedIndexes"` Preview feature enabled, the signature is:
+> **Note**: In versions `4.0.0` and later, and in versions `3.5.0` and later with the `extendedIndexes` Preview feature enabled, the signature is:
```
Comment (janpio) [inline content/400-reference/200-api-reference/100-prisma-schema-reference.mdx]: I think this might actually justify switching the order around:
First the new signature, then the one from before 4.0.0 (or between 3.5.0 and 4.0.0 without `extendedIndexes`).
We should list the new default first, and then list the special cases from the past.

(Same for the other changes in this file)

File: content/400-reference/300-database-reference/01-database-features.mdx [hunk 11]
```diff
@@ -60,20 +60,23 @@ This section describes which database features exist on the relational databases
 | -------- | ---------- | -------------------- | ----- | ------ | ----------- | ------------------------------------------------------------------------------------------------- | ------------- | -------------- |
 | `UNIQUE` | ✔️         | ✔️                   | ✔️    | ✔️     | ✔️          | [`@unique` and `@@unique`](/concepts/components/prisma-schema/data-model#defining-a-unique-field) | ✔️            | ✔️             |
 | `WHERE`  | ✔️         | ✔️                   | ✔️    | ✔️     | ✔️          | Not yet                                                                                           | ✔️            | Not yet        |
-| `USING`  | ✔️         | ✔️                   | ✔️    | ✔️     | ✔️          | Not yet                                                                                           | ✔️            | Not yet        |
+| `USING`  | ✔️         | ✔️                   | ✔️    | ✔️     | ✔️          | ✔️†                                                                                               | ✔️            | Not yet        |
```
Comment (janpio) [inline content/400-reference/300-database-reference/01-database-features.mdx:63]: I think this should link to the way this an be used, so `type` I guess? Otherwise I have no context on how I can actually make this happen in Prisma.

(I note this is not happning in most of the table - this could potentially happen later via a separate issues @keerlu )

(Comment also applies for other change in this file)

File: content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx (hunk 8 above)
Comment (janpio) [reply content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx:291]: Look a bit like auto formatter things. Comments before attribute etc 🤷

File: content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx (hunk 2 above)
Comment (keerlu) [reply content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx]: Added a link to the upgrade section of the page which gives more detail

File: content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx (hunk 8 above)
Comment (keerlu) [reply content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx:291]: Yes it's an autoformatter thing - I've put in prettier ignore markings

File: content/400-reference/200-api-reference/100-prisma-schema-reference.mdx [hunk 12]
```diff
@@ -1429,11 +1429,12 @@ Defines a multi-field ID (composite ID) on the model.
...
+|-------------|----------|--------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `fields`    | **Yes**  | `FieldReference[]` | A list of field names - for example, `["firstname", "lastname"]`                                                                                      |
+| `name`      | **No**   | `String`           | The name that the Client API will expose for the argument covering all fields, e.g. `fullName` in `fullName: { firstName: "First", lastName: "Last"}` |
+| `map`       | **No**   | `String`           | The name of the underlying primary key constraint in the database                                                                                     |
+| `clustered` | **No**   | `Boolean`          | **SQL Server only.** Defines whether the index is clustered or non-clustered. Defaults to `true`.                                                            |
```
Comment (janpio) [inline content/400-reference/200-api-reference/100-prisma-schema-reference.mdx]: Formatting only:
```suggestion
| `clustered` | **No**   | `Boolean`          | **SQL Server only.** Defines whether the index is clustered or non-clustered. Defaults to `true`.                                                     |
```

File: content/400-reference/200-api-reference/100-prisma-schema-reference.mdx [hunk 13]
```diff
@@ -2378,11 +2383,8 @@ Defines an index in the database.
 - It is currently not possible to provide more configuration options to the index:
```
Comment (janpio) [inline content/400-reference/200-api-reference/100-prisma-schema-reference.mdx]:
```suggestion
- There are some additional index configuration options that cannot be provided via the Prisma schema yet. These include:
```

**Review by janpio — APPROVED** [review]
👍 (besides existing comments)

File: content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx (hunk 7 above)
Comment (keerlu) [reply content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx:11]: Yes this is intentional, I'd rather give the information in easier-to-read form further down rather than including it in the TopBlock

File: content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx (hunk 5 above)
Comment (keerlu) [reply content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx]: Have now linked to the upgrade guide and made it clear that this warning is for earlier versions

File: content/400-reference/200-api-reference/100-prisma-schema-reference.mdx (hunk 10 above)
Comment (keerlu) [reply content/400-reference/200-api-reference/100-prisma-schema-reference.mdx]: Agree, I've moved the signatures around

File: content/400-reference/300-database-reference/01-database-features.mdx (hunk 11 above)
Comment (keerlu) [reply content/400-reference/300-database-reference/01-database-features.mdx:63]: Made an issue for remaining changes like this: https://github.com/prisma/docs/issues/3332

File: content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx (hunk 6 above)
Comment (keerlu) [reply content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx:27]: Agree that this could do with reorganisation, made an issue to look at this after the release: https://github.com/prisma/docs/issues/3332

File: content/200-concepts/100-components/250-preview-features/050-client-preview-features.mdx [hunk 14]
```diff
@@ -11,6 +11,7 @@ The following [Preview](/about/prisma/releases#preview) feature flags are availa
 - [`interactiveTransactions`](/concepts/components/prisma-client/transactions#interactive-transactions-in-preview) since release [2.29.0](https://github.com/prisma/prisma/releases/tag/2.29.0) - [Submit feedback](https://github.com/prisma/prisma/issues/8664)
 - [`fullTextSearch`](/concepts/components/prisma-client/full-text-search) since release [2.30.0](https://github.com/prisma/prisma/releases/tag/2.30.0) - [Submit feedback](https://github.com/prisma/prisma/issues/8877)
 - [`referentialIntegrity`](/concepts/components/prisma-schema/relations/referential-integrity) since release [3.1.1](https://github.com/prisma/prisma/releases/tag/3.1.1) - [Submit feedback](https://github.com/prisma/prisma/issues/9380)
+- [`dataProxy`](/concepts/data-platform/data-proxy) since release [3.3.0](https://github.com/prisma/prisma/releases/tag/3.3.0) - [Submit feedback](https://github.com/prisma/prisma/issues/9841)
```
Comment (janpio) [inline content/200-concepts/100-components/250-preview-features/050-client-preview-features.mdx:14]: I think this somehow popped up here again but should not.

(Makes me worry there are other things)

**Review by janpio — CHANGES_REQUESTED** [review]
see comments with suggestions

File: content/200-concepts/100-components/250-preview-features/050-client-preview-features.mdx [hunk 15]
```diff
@@ -11,6 +11,7 @@ The following [Preview](/about/prisma/releases#preview) feature flags are availa
...
 - [`extendedIndexes`](/concepts/components/prisma-schema/indexes#index-configuration) since release [3.5.0](https://github.com/prisma/prisma/releases/tag/3.5.0) - [Submit feedback](https://github.com/prisma/prisma/issues/10540)
```
Comment (janpio) [inline content/200-concepts/100-components/250-preview-features/050-client-preview-features.mdx:15]: I think this should also be removed now, or is that done in another PR?

File: content/400-reference/200-api-reference/100-prisma-schema-reference.mdx [hunk 16]
```diff
@@ -2174,20 +2179,20 @@ These arguments are added to the relevant field names:
...
+> @@unique(_ fields: FieldReference[], name: String?, map: String?)
 > ```
 
-> **Note**: In version 3.5.0 and later, the signature for the `"extendedIndexes"` Preview feature is:
+> **Note**: Before version 3.0.0, the signature was:
 >
 > ```prisma no-lines
-> @@unique(_ fields: FieldReference[], name: String?, map: String?)
+> @@unique(_ fields: FieldReference[], name: String?)
 > ```
```
Comment (janpio) [inline content/400-reference/200-api-reference/100-prisma-schema-reference.mdx:2196]: Is the end result correct here? Seems a bit like it is all in a quote now.
(See all the lines above until `#### Signature`)

File: content/200-concepts/100-components/250-preview-features/050-client-preview-features.mdx (hunk 15 above)
Comment (janpio) [reply content/200-concepts/100-components/250-preview-features/050-client-preview-features.mdx:15]:
```suggestion
```

File: content/200-concepts/100-components/250-preview-features/050-client-preview-features.mdx (hunk 14 above)
Comment (janpio) [reply content/200-concepts/100-components/250-preview-features/050-client-preview-features.mdx:14]:
```suggestion
```

File: content/400-reference/200-api-reference/100-prisma-schema-reference.mdx (hunk 16 above)
Comment (janpio) [reply content/400-reference/200-api-reference/100-prisma-schema-reference.mdx:2196]: No, I think this looks weird in the rendered view here on GH:
image

Comment (janpio) [reply content/400-reference/200-api-reference/100-prisma-schema-reference.mdx:2196]: Here is the preview, also a bit broken I think:
image

File: content/200-concepts/100-components/250-preview-features/050-client-preview-features.mdx (hunk 14 above)
Comment (janpio) [reply content/200-concepts/100-components/250-preview-features/050-client-preview-features.mdx:14]: (I could not find any other things that got weird via the merge)

File: content/400-reference/200-api-reference/100-prisma-schema-reference.mdx [hunk 17]
```diff
@@ -2174,20 +2179,20 @@ These arguments are added to the relevant field names:
 
 #### Signature
 
-```prisma no-lines
-@@unique(_ fields: FieldReference[], name: String?, map: String?)
-```
+> ```prisma no-lines
+> @@unique(_ fields: FieldReference[], name: String?, map: String?)
+> ```
```
Comment (janpio) [inline content/400-reference/200-api-reference/100-prisma-schema-reference.mdx:2184]:
````suggestion
```prisma no-lines
@@unique(_ fields: FieldReference[], name: String?, map: String?)
```
````

**Review by janpio — APPROVED** [review]
Ignore previous blocking comment.
